src/Context/UsersContext.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const UsersContext = React.createContext()
+
+export default UsersContext
```

Comment:
I like the idea of creating a context folder with separate context files. Not sure if it would be better to have all our contexts in one file, but I assume this is the best practice for the long term.